core/src/main/java/org/testcontainers/dockerclient/DockerClientProviderStrategy.java

```diff
@@ -240,7 +240,7 @@ public static DockerClient getClientForConfig(TransportConfig transportConfig) {
         DefaultDockerClientConfig.Builder configBuilder = DefaultDockerClientConfig.createDefaultConfigBuilder();
 
         if (configBuilder.build().getApiVersion() == RemoteApiVersion.UNKNOWN_VERSION) {
-            configBuilder.withApiVersion(RemoteApiVersion.VERSION_1_30);
+            configBuilder.withApiVersion(RemoteApiVersion.VERSION_1_34);
```

Comment:
Not sure the best approach here but platform query param was only introduced in v1.34 and locally on my M1 Mac it is always returning unknown version and defaulting to v1.30 which did not have it.

Reply:
IIRC this is because the version of the docker API is newer than any version that docker-java knows about, so it's identifying it as 'unknown'. I think we should really tackle that in docker-java, and then there should be no need to make this change.

Reply:
Ah brilliant, that makes sense. It did feel somewhat wrong doing it.

Reply:
@dameikle do we still need this change? This would up the Docker version requirement - not something that we would do in general.